tests/wpt/meta/content-security-policy/generic/wildcard-host-part.sub.window.js.ini

```diff
@@ -0,0 +1,2 @@
+[wildcard-host-part.sub.window.html]
+  expected: CRASH
```

Comment:
Is this a flake maybe?

Reply:
Shouldn't be. It's been consistent each time I've run it.